src/test/java/HW17.java

```diff
@@ -0,0 +1,22 @@
+
+import org.testng.annotations.Test;
+
+import java.time.Duration;
+
+
+public class HW17 extends MyBaseTest {
```

Comment:
Please, avoid own names or vague words: If there’s not a real need for them, avoid using these words, which often add no value

For exampe

`class HomeWork17`